### ISSUE TITLE
Seller experience: Consider reducing the whitespace in the mobile view.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#61020]
We currently have a lot of whitespace for the home page in the mobile view. I'm assuming this is related to the Zoologist theme itself, but let's also make sure we don't have extra elements taking up space in the commerce pattern and the homepage template.

![2022-02-10 12 58 18](https://user-images.githubusercontent.com/349751/153495594-ef5b5d15-d5fe-4b33-b99f-56a96c37866a.gif)

**Comment by kwight** [comment]
@SaxonF @alaczek Any guidance here?

**Comment by kwight** [comment]
This is what it looks like now, after the recent pattern changes and work. It's still a lot of whitespace at the top.

![2022-02-17 13 36 55](https://user-images.githubusercontent.com/349751/154575295-07ca68e7-b8e1-4a17-bc6c-acc8ea380546.gif)

**Comment by alaczek** [comment]
I agree the amount of whitespace on mobile is excessive. I updated the pattern to use the vh units and it helped, but what I'd really like to tackle is the while space above the pattern in the header. Blockbase comes with a few header patterns to choose from:
![image](https://user-images.githubusercontent.com/4550351/154588597-72088dae-89a3-4acc-b4b6-805dc4cfe8ae.png)


Would it be at all possible to switch to the "Header Linear" header pattern by default? This is what I did on my test site, and it's looking much better: https://kittycakes814502261.wordpress.com/
